datacenter/dtr/2.4/guides/admin/monitor-and-troubleshoot/notary-audit-logs.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Check Notary audit logs
+description: When you push signed images, Docker Trusted Registry keeps audit
+  logs for the changes made to the image metadata. Learn how to view these logs.
+keywords: registry, monitor, troubleshoot
+---
+
+DTR keeps audit logs of changes made to trusted repositories.
```

Comment:
This should say "Docker Content Trust keeps audit logs..." since this feature is specifically about trusted repositories.  There is a different feature (which can be accessed at the events API endpoint) which lets you view audit information in DTR, but it's not cryptographically signed.  The same goes below here where you say "DTR logs that information" should really be "Docker Content Trust logs that information".